exercises/transpose/examples/success-standard/src/Transpose.hs

```diff
@@ -0,0 +1,22 @@
+{-# LANGUAGE LambdaCase #-}
+
+module Transpose (transpose) where
+
+import Data.Maybe (isNothing)
+
+transpose :: [String] -> [String]
+transpose [] = []
+transpose rows =
+  map toString $ foldr 
+    (zipWith (:) . padRight . map Just) 
+    (replicate longestLength []) 
+    rows
+  where 
+    longestLength = maximum $ map length rows
+    padRight line = line ++ replicate (longestLength - length line) Nothing
+    toString line = concatMap (\case Nothing -> " "
+                                     Just c -> c:"") $ dropRight line
+    dropRight line = snd $ foldr 
+          (\x (ignore, xs) -> if ignore && isNothing x 
+                                then (ignore, xs) 
+                                else (False, x:xs)) (True,[]) line
```

Comment:
In the `(True,[])` there is no space after the comma, but there is a space after all other commas in this file. I think for the purpose of consistency it would be good to have a comma after the space too.

Reply:
Good call!

Reply:
Actually, due to `dropWhileEnd`, this entirely goes away.